src/sentry/static/sentry/app/components/stackedBarChart.jsx

```diff
@@ -42,73 +42,42 @@ const StackedBarChart = createReactClass({
     ),
     tooltip: PropTypes.func,
     barClasses: PropTypes.array,
-  },
-
-  statics: {
-    getInterval(series) {
-      // TODO(dcramer): not guaranteed correct
-      return series.length && series[0].data.length > 1
-        ? series[0].data[1].x - series[0].data[0].x
-        : null;
-    },
-
-    pointsToSeries(points) {
-      let series = [];
-      points.forEach((p, pIdx) => {
-        p.y.forEach((y, yIdx) => {
-          if (!series[yIdx]) {
-            series[yIdx] = {data: []};
-          }
-          series[yIdx].data.push({x: p.x, y});
-        });
-      });
-      return series;
-    },
-
-    pointIndex(series) {
-      let points = {};
-      series.forEach(s => {
-        s.data.forEach(p => {
-          if (!points[p.x]) {
-            points[p.x] = {y: [], x: p.x};
-          }
-          points[p.x].y.push(p.y);
-        });
-      });
-      return points;
-    },
-  },
-
-  getDefaultProps() {
-    return {
-      className: 'sparkline',
-      height: null,
-      label: '',
-      points: [],
-      series: [],
-      markers: [],
-      width: null,
-      barClasses: ['chart-bar'],
-    };
-  },
+    minHeights: PropTypes.arrayOf(PropTypes.number),
```

Comment:
Can we document what `minHeights` does? It's not clear why it's an array of min heights.

Reply:
I also considered renaming it to `minBarHeights` which might help